olp-cpp-sdk-dataservice-read/src/generated/api/QueryApi.h

```diff
@@ -113,6 +114,13 @@ class QueryApi {
       boost::optional<std::string> billing_tag,
       client::CancellationContext context);
 
+  static olp::client::HttpResponse CreateQuadTreeIndex(
+      const client::OlpClient& client, const std::string& layer_id,
+      int64_t version, const std::string& quad_key, int32_t depth,
+      boost::optional<std::vector<std::string>> additional_fields,
+      boost::optional<std::string> billing_tag,
+      client::CancellationContext context);
+
```

Comment:
`CreateQuadTreeIndex` is a very unsuitable name. Are you creating the quadtree index yourself or are you requesting it from the platform?
Name it QuadTreeIndex() with boost::optional<int64_t> as version and then we can remove both others.
Return type HttpResponse is good.

Reply:
Done, but others QuadTreeIndex() are still there. Replacing them requires many corrections in *Repository classes and tests. Probably, need separate ticket for that.

Reply:
Yes, we should do this in a separate ticket.